kolibri/plugins/coach/assets/src/routes/index.js

```diff
@@ -26,7 +26,8 @@ export default [
     path: '/classes',
     component: CoachClassListPage,
     handler(toRoute) {
-      store.dispatch('loading');
+      // loading state is handled locally
+      store.dispatch('notLoading');
```

Comment:
This accords with my general sense that we can reduce the role of the core loading state to be about toggling the linear loader when the navigation is still resolving. In this frame, we could handle it entirely within our core routing setup, where we toggle it on before route enter, and toggle it after after the route has resolved.